Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper.json

```diff
@@ -1,101 +1,276 @@
+                    "simple": "hosts.machine_domain"
+                },
+                "Start Time": {
+                    "simple": "start"
+                },
+                "State": {
+                    "simple": "state"
+                },
+                "Tactic": {
+                    "simple": "tactics"
+                },
+                "Technique": {
+                    "simple": "techniques"
+                },
+                "Users": {
+                    "simple": "users"
+                },
+                "name": {
+                    "complex": {
+                        "filters": [],
+                        "root": "incident_id",
+                        "transformers": [
+                            {
+                                "args": {
+                                    "prefix": {
+                                        "value": {
+                                            "simple": "Falcon Incident - ID: "
+                                        }
+                                    },
+                                    "suffix": {}
+                                },
+                                "operator": "concat"
+                            }
+                        ]
+                    }
+                },
+                "severity": {
+                    "complex": {
+                        "filters": [],
+                        "root": "fine_score",
+                        "transformers": [
+                            {
+                                "args": {
+                                    "by": {
+                                        "value": {
+                                            "simple": "50"
+                                        }
+                                    }
+                                },
+                                "operator": "division"
+                            },
+                            {
+                                "args": {
+                                    "by": {
+                                        "value": {
+                                            "simple": "2"
+                                        }
+                                    }
+                                },
+                                "operator": "addition"
+                            },
+                            {
+                                "operator": "floor"
+                            }
+                        ]
+                    }
+                }
+            }
+        },
+        "dbot_classification_incident_type_all": {
+            "dontMapEventToLabels": false,
+            "internalMapping": {
+                "Detected User": {
+                    "simple": "users"
+                },
+                "OS Version": {
+                    "simple": "device.os_version"
+                },
+                "Scenario": {
+                    "simple": "behaviors.[0].scenario"
```

Comment:
Is This deliberate ?

Reply:
Nope you are right - should be without the index [0]